### PR TITLE
adds the ancient spear as a null rod option

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

adds the ancient spear as a new null rod option, representing the one antagman missing (clockwork cult)
also makes hypertool path name not capitalized because thats dumb and moves it from the middle of the scythe section

## Why It's Good For The Game

clockies were the one thing missing from chaps null rod, clock sprites are great, would seem to be a fun option to have a spear, also gives ratvar deity chaplains a weapon to pick

## Changelog
:cl:
add: Adds a new null rod option, the ancient spear.
/:cl:

### CODE REVIEW

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -660,3 +645,32 @@
...
+	icon_state = "ratvarian_spear"
+	item_state = "ratvarian_spear"
+	lefthand_file = 'icons/mob/inhands/antag/clockwork_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/antag/clockwork_righthand.dmi'
+	icon = 'icons/obj/clockwork_objects.dmi'
+	slot_flags = ITEM_SLOT_BELT
+	armour_penetration = 10
+	sharpness = IS_SHARP_ACCURATE
+	w_class = WEIGHT_CLASS_BULKY
+	attack_verb = list("stabbed", "poked", "slashed")
```
Comment (Dennok) [inline code/game/objects/items/holy_weapons.dm]: Maybe add "clocked" here?

Comment (Fikou) [reply code/game/objects/items/holy_weapons.dm]: sounds nice i added it

Comment (Dennok) [reply code/game/objects/items/holy_weapons.dm]: Thanks.

**Comment by ExcessiveUseOfCobblestone** [comment]
old cult tome next please

**Comment by fludd12** [comment]
Isn't that already a bible type option?

**Comment by ExcessiveUseOfCobblestone** [comment]
idk

**Comment by OverlyGeneric** [comment]
how does this work exactly?

does it have the original force values of the item?

does it disappear once you have thrown and hit it with someone?

**Comment by Fikou** [comment]
> how does this work exactly?
> 
> does it have the original force values of the item?
> 
> does it disappear once you have thrown and hit it with someone?

bruh just read the code its a null rod variant
i gave it armour penetration, sharpness, look and sound of the ratvarian spear, otherwise its the same as every rod

**Comment by ExcessiveUseOfCobblestone** [comment]
pls use ur buzzwords properly

**Comment by Fikou** [comment]
> pls use ur buzzwords properly

no

**Comment by Shadowflame909** [comment]
Lower its attack force but make it do some damn good embed damage.

Thats a source of damage that needs more use

**Comment by Fikou** [comment]
if i made it an embed weapon it would just be a better version of the fedora

**Pull request merged**